src/components/PurchaseButton.astro

```diff
@@ -17,15 +17,17 @@ import Button from "./Button.astro"
     if (referrer) {
         const campaignUrl = {
             // Twitter
-            tw: "https://pretix.eu/pyconau/2023/c/kZnF7EnvL/",
+            tw: "https://pretix.eu/pyconau/2024/c/V6nsZVR5k/",
             // LinkedIn
-            li: "https://pretix.eu/pyconau/2023/c/QsEV6PAEj/",
+            li: "https://pretix.eu/pyconau/2024/c/Z4m7ctrPC/",
             // Fediverse
-            fv: "https://pretix.eu/pyconau/2023/c/zEotuIaUC/",
+            fv: "https://pretix.eu/pyconau/2024/c/1mPdQLtHo/",
+
+            // Uncomment and update these values if we end up paying for ads. 
             // Twitter Ads
-            ta: "https://pretix.eu/pyconau/2023/c/PXCb4Blaj/",
+            // ta: "https://pretix.eu/pyconau/2023/c/PXCb4Blaj/",
```

Comment:
I'd argue that these could be deleted, maybe can leave a comment if we really want to remember this, but these are pretty unlikely and links obviously old.